Remove OIDC and SAML adapters for Wildfly/EAP ZIP downloads. Update documentation. (#24877)
keycloak/keycloak: docs/documentation/securing_apps/topics/oidc/java/jboss-adapter.adoc

[[_jboss_adapter]]

==== JBoss EAP/WildFly adapter

{project_name} provided this adapter in the past, but it is not provided anymore.

We recommend that you switch to the Elytron OIDC library to secure your applications.
This library has a similar configuration to the {project_name} WildFly adapters, so you can expect a smooth migration of your applications
if you used this adapter with the older {project_name} versions.

Elytron OIDC library works with WildFly 28 or newer versions. For the older WildFly versions or for JBoss EAP 7, it is recommended to upgrade
to newer WildFly/EAP or look for some alternative OIDC client library. Otherwise, you will need to stick with the older {project_name} adapters, but those
are not maintained and officially supported.

> For more details about how to integrate {project_name} with JakartaEE applications running on latest Wildfly/EAP, consider looking at the
> {quickstartRepo_link}[Keycloak Quickstart GitHub Repository] at the jakarta quickstarts.

Reviewer: ```suggestion
For more details on how to integrate {project_name} with JakartaEE applications running on latest Wildfly/EAP, take a look at the Jakarta EE quickstarts within the {quickstartRepo_link}[Keycloak Quickstart GitHub Repository].
```
